core/src/main/scala/za/co/absa/spline/harvester/dispatcher/HttpLineageDispatcher.scala

```diff
@@ -61,20 +98,41 @@ class HttpLineageDispatcher(splineServerRESTEndpointBaseURL: String, http: BaseH
 
   private def sendJson(json: String, url: String) = {
     log.debug(s"sendJson $url : $json")
-    try http(url)
-      .postData(json)
-      .compress(true)
+
+    try {
+      val http = if (useRequestCompression) {
```

Comment:
nit: try to keep `if` and `else` keywords vertically aligned. It's much easier to read.
In the current code block `else` is aligned with `val`, rather then `if`.   
E.g.
```scala
val http = 
  if (...) {
    ...
  } else { // aligned with 'if'
    ...
  } 
```
vs
```scala
val http = if (...) {
    ...
  } else { // aligned with 'http'
    ...
  } 
```